pkg/sql/catalog/catconstants/constants.go

```diff
@@ -30,6 +30,7 @@ const InternalAppNamePrefix = ReportableAppNamePrefix + "internal"
 // DelegatedAppNamePrefix should be scrubbed in reporting.
 const DelegatedAppNamePrefix = "$$ "
 
+const InternalSqlAppName = "cockroach sql"
```

Comment:
nit: any public constant like this needs a comment. the comment has to start with the name of the constant (see above for example)

so like
```
InternalSqlAppName is the application_name used by the cockroach CLI by default
```